src/cft/cft.jl

```diff
@@ -0,0 +1,162 @@
+function supporter_functions(λ::Number, μ::Number, η::Number; W::Real=1000., δ::Real=0.001, 
+    s::AbstractVector=[1.0], N::Int=5, stabilise::Bool=false)
+    
+    ω=range(-W, W, step=δ)
+    # ω=range(-W, W, length=2000000); δ = step(ω);
+    ω = ω[1:end-1]
+
+    # FIXME: Should probably find a nicer way around this
+    if μ ≈ 0 && real(λ) < 0 && λ ∈ ω
+        λ += 1e2*eps() * π
+        @warn "μ ≈ 0, λ < 0, and λ ∈ Sample, we slightly perturb λ to avoid NaNs in the FFT computation in supporter_functions in cft.jl"
+    end
+
+    fmultiplier = k -> (λ .- im.*μ.*sign.(k) .+im.*η.*k .+ abs.(k))

```

Comment:
Don't broadcast (also below):
```suggestion
fmultiplier = k -> (λ - im*μ*sign(k) +im*η*k + abs(k))
```